packages/dashboard/src/components/dashboard/index.test.tsx

```diff
@@ -66,6 +66,47 @@ it('renders in readonly initially', function () {
   expect(queryByTestId(/time-selection/i)).toBeInTheDocument();
 });
 
+it('passes the correct viewMode to onSave', function () {
```

Comment:
Could you add a test to test the initial view mode without setting `viewMode`?

Reply:
added!